config/resources/organisation.lisp

```diff
@@ -11,7 +11,8 @@
                 (:alternatieve-naam :string-set ,(s-prefix "skos:altLabel"))
                 (:wil-mail-ontvangen :boolean ,(s-prefix "ext:wilMailOntvangen")) ;;Voorkeur in berichtencentrum
                 (:mail-adres :string ,(s-prefix "ext:mailAdresVoorNotificaties"))
-                (:is-trial-user :boolean ,(s-prefix "ext:isTrailUser")))
+                (:is-trial-user :boolean ,(s-prefix "ext:isTrailUser"))
+                (:has-view-only-module :string-set ,(s-prefix "ext:hasViewOnlyModule")))
```

Comment:
nitpick: I would call this something like `view-only-modules` since it contains the actual list of module names.